samples/openthread/cli/sample.yaml

```diff
@@ -1,77 +1,113 @@
 sample:
   name: OpenThread CLI sample
   description: Test OpenThread Command Line Interface.
+
 tests:
-  sample.openthread.cli:
+  sample.openthread.cli.multiprotocol:
     build_only: true
-    platform_allow: >
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840
-      nrf52840dongle_nrf52840 nrf21540dk_nrf52840
```

Comment:
I think we should keep `nrf52840dongle_nrf52840` targets?

Reply:
nrf52840dongle_nrf52840 back in platform_allow